comments/classified.csv

```diff
@@ -1820,7 +1806,6 @@ I have no faith in you,1,0.5
 I have no faith in this code,1,0.5
 "I like the way you code, you got this",0,1
 I have no faith in this at all,1,0.5
-#url,0,0.5
```

Comment:
I think we should keep at least one of these?

Reply:
Fixed!